multicore_runtime/wsgi.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Configure a user project and instantiate WSGI app meta_app to serve it.
+
+Importing this module will result in side-effects, such as registering the
+project's default ticket via vmstub.Register. This is a broadly compatible way
+to induce the webserver to run initialization code before starting to serve the
+WSGI app.
+"""
+
+import logging
+import os
+
+import cloud_logging
+from dispatcher import dispatcher
```

Comment:
Why not:

```python
  import dispatcher
  import middleware
  import wsgi_config
```

and then refer to the functions by their module-qualified names (`wsgi_config.env_vars_from_env_config`, etc.)?